src/actions/teams.ts

```diff
@@ -299,7 +299,7 @@ export function getMyTeamMembers(): ActionFunc {
     };
 }
 
-export function getTeamMembers(teamId: string, page = 0, perPage: number = General.TEAMS_CHUNK_SIZE): ActionFunc {
+export function getTeamMembers(teamId: string, page = 0, perPage: number = General.TEAMS_CHUNK_SIZE, options = {}): ActionFunc {
```

Comment:
I would opt for creating a type `GetTeamMembersOpts` instead of `options = {}` 
something like 
```
type GetTeamMembersOpts = {
    sort: 'Username' | undefined;
    exclude_deleted_users: bool | undefined;
}
```

Reply:
Also 'Username' and 'UserId' should be constants.

Reply:
I dont know if constants can be used in a type definition i seem to remember having issues with that

Reply:
Thanks for the suggestion! 
@sbishel I did make Username a constant since I do use it in the webapp however seems like you cannot use it within a type definition. Thanks for the suggestion.

@fm2munsh I did use the type you recommended however I did not include 'UserId' as that is default behaviour when you don't pass anything to sort.